include/Network/HTTP/BadGatewayException.php

```diff
@@ -9,6 +9,7 @@
 /**
  * @brief Bad Gateway (502) http exception
  */
-class BadGatewayException extends HTTPException {
+class BadGatewayException extends HTTPException 
+{
```

Comment:
I think this isn't really decided at the moment. As far as I can remember @Hypolite said it is part of the PSR standard.